listener/listener.go

```diff
@@ -164,6 +164,32 @@ func (l *Listener) Listen(ctx context.Context, handler message.Handler, topicNam
 	return listenerHandle.Err()
 }
 
+// DeleteSubscriptions deletes the giving subscriptions
```

Comment:
I don't think that it is a good API to provide "Delete an arbitrary subscription name" from a listener to a different subscription name. Listener is to listen.
The api is available at a lower level, on the "subscription manager", via the namespace.
that should be enough

Reply:
okay, I am also okay to use the api directly, created a separate pr for it. closing this one.